WordPress/Classes/Models/Notifications/Notification.swift

```diff
@@ -365,18 +365,21 @@ extension Notification {
     }
 
     var allAvatarURLs: [URL] {
-        let firstMedias: [AnyObject] = body?.compactMap {
-            let allMedia = $0["media"] as? [AnyObject]
-            return allMedia?.first as? AnyObject
-        } ?? []
-
-        let urlStrings = firstMedias.compactMap {
-            $0["url"] as? String
+        let users = body?.filter({ element in
```

Comment:
I think we can add another unit test to `NotificationTests` below `testAllAvatarURLsReturnMultipleURLs` by replicating `notifications-like-multiple-avatar.json` and test if an item with different type will be caught in the process. Wdyt?

Reply:
@alpavanoglu I updated `notifications-like-multiple-avatar.json` to include non-user media. This means `testAllAvatarURLsReturnMultipleURLs` covers this scenario. Can you elaborate on what another test would achieve?

Reply:
> test that checks the count when there's a non-user media

The existing test covers this. I added non-user media to the `notifications-like-multiple-avatar.json` file. So the notification now has a total of 4 media, 3 user media, and 1 non-user. The test checks that the media parsed (allAvatarURLs.count) is equal to 3. Meaning it checks that the non-user media is filtered.